### DISCUSSION TITLE
blog/programming-tasks

### DISCUSSION THREAD

**Discussion by giscus** [GawdOfGithub/dcc-blog#8] [General]
# blog/programming-tasks

https://blog.dccnita.in/blog/programming-tasks

**Comment by Biprajit-Palit** [comment]
I have written the coding problems in MS word ...so to upload it...should I export it to plain text file and submit in g drive or exporting it to pdf will be enough?

**Reply by VijetaPriya47** [reply]
Text (.txt) files are preferable. Docs and Word files are also fine. PDFs are not recommended at all.
